src/transformers/keras_callbacks.py

```diff
@@ -63,9 +63,9 @@ def rouge_fn(predictions, labels):
             supplied.
         batch_size (`int`, *optional*):
             Batch size. Only used when the data is not a pre-batched `tf.data.Dataset`.
-        predict_with_generate (`bool`, *optional*, defaults to `False`):
+        predict_with_generate (`bool`, defaults to `False`):
             Whether we should use `model.generate()` to get outputs for the model.
-        use_xla_generation (`bool`, *optional*, defaults to `False`):
```

Comment:
The optional should still be there in the docstring because the argument is optional in the sense it has a default value. It's just the type annotation `Optional` (which means `Union None`) that wasn't correct ;-)

Reply:
Ah, I never knew that was our rule for writing docstrings! Fixing.